test/message/events_unhandled_error_common_trace.out

```diff
@@ -5,18 +5,8 @@ node:events:*
 Error: foo:bar
     at bar (*events_unhandled_error_common_trace.js:*:*)
     at foo (*events_unhandled_error_common_trace.js:*:*)
-    at Object.<anonymous> (*events_unhandled_error_common_trace.js:*:*)
-    at Module._compile (node:internal/modules/cjs/loader:*:*)
-    at Module._extensions..js (node:internal/modules/cjs/loader:*:*)
-    at Module.load (node:internal/modules/cjs/loader:*:*)
-    at Module._load (node:internal/modules/cjs/loader:*:*)
-    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:*:*)
-    at node:internal/main/run_main_module:*:*
 Emitted 'error' event at:
     at quux (*events_unhandled_error_common_trace.js:*:*)
     at Object.<anonymous> (*events_unhandled_error_common_trace.js:*:*)
-    at Module._compile (node:internal/modules/cjs/loader:*:*)
-    [... lines matching original stack trace ...]
```

Comment:
If it's the only test checking this line, maybe we should also move it to a JS test with RegExp.